### ISSUE TITLE
The preprocess code does not generate all the label files 

### ISSUE THREAD

**Issue by Shrutika994** [ymli39/DeepSEED-3D-ConvNets-for-Pulmonary-Nodule-Detection#7]
[](url)
Hello, I'm getting the following issue. I preprocessed the images but it happens not to generate a certain amount of label files which creates a problem while training. Would it be possible to look into this please?
![send to github](https://user-images.githubusercontent.com/38447988/63778196-57758700-c91f-11e9-83c3-483fac33e79b.PNG)

**Comment by ymli39** [comment]
Did you check the name of the labels it generated instead?

**Comment by Shrutika994** [comment]
Yes I checked the other labels it generated for the LUNA16 dataset but, while training, this problem seem to arise every time, so model can't be trained. So would it be possible that it takes the other label files as input for training and if it is possible could you please explain how.

**Comment by ymli39** [comment]
Does the npy file exist in the directory or it is not generated. If the npy already exists, I recommand you sent a debug point there to see why it does not load.

**Comment by Shrutika994** [comment]
The npy file is not generated.

**Comment by ymli39** [comment]
I could generate all labels on my machine with no problem. I would recommend make a csv file that only contains label 360 to see the preprocessing results.

**Comment by Shrutika994** [comment]
Okay I will try doing that. Thanks.

**Comment by MjdMahasneh** [comment]
@Shrutika994 @ymli39 

Could you guys please help me understand the parameter luna_abbr in config_training.py
I am trying to prepare LUNA-16 but am I can't find out what/where luna_abbr file is?

From what I see in the config_training.py, luna_abbr seems to be pointing at a file called shorter.csv, which I can't find nor can I tell how to produce it.

Any help would be very much appreciated :)

**Comment by MjdMahasneh** [comment]
@Shrutika994 

The paper mentions 10-folds validation (which I presume, also means there is 10-fold training), my question is :

1-how is the 10-fold training done using the training script?
2-are these files (luna_train.npy and luna_test.npy) related to the 10-fold training?
3-how are these files generated?

Looking forward to hearing back from you.
